aiotruenas_client/websockets/protocol.py

```diff
@@ -121,6 +117,9 @@ async def subscribe(self, name: str) -> asyncio.Queue:
         )
         return await sub_future
 
+    async def _authenticate(self):
+        raise NotImplementedError
```

Comment:
We should use `@abstractmethod` here instead (and possibly include `ABC` if it's not already in the chain).